src/test/unit_tests/test_bucketspace.js

```diff
@@ -478,12 +641,22 @@ mocha.describe('bucket operations - namespace_fs', async function() {
         assert.ok(list_ok);
     });
     mocha.it('list buckets with dn', async function() {
-        // Give s3_correct_uid access to the required buckets
         const res = await s3_correct_dn_default_nsr.listBuckets({});
         console.log(inspect(res));
-        const list_ok = bucket_in_list([bucket_name + "-s3-2"], [], res.Buckets);
+        const list_ok = bucket_in_list([bucket_name + '-s3-2'], [], res.Buckets);
         assert.ok(list_ok);
     });
+
+    mocha.it('delete bucket with dn', async function() {
```

Comment:
Why did you add this test?

Reply:
why not?
this bucket was not cleaned on the teardown, so instead of removing it on teardown I added a test that checks also that we can delete this bucket using dn

Reply:
I asked because I didn't understand how this test is related to the list buckets.
As I understand it now it is not related to the change, but you added a test to improve our coverage with bucket deletion using a distinguished name.